src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    /**
+     * main functions like a to-do list
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        String input = "";
+        Boolean end = false;
```

Comment:
Perhaps use a name that sounds more like a boolean? What about something like 'isEnd' or 'isExited'?